src/scripts/init.sh

```diff
@@ -77,8 +79,8 @@ echo "Schema created or already exists."
 # Pull the meters into the DB and get data
 if [ $1 != "NONE"  ]; then
     set -e
-    echo "Trying to add meters."
-    npm run addMamacMeters $1 2> /dev/null
+    echo "Trying to add meters from '$IPS_FILE'."
+    npm run addMamacMeters $IPS_FILE 2> /dev/null
```

Comment:
Just curious, how does this fix the issue? Aren't you just aliasing $1 to IPS_FILE?

Reply:
Yes. The way I parse arguments causes arg 2 to be shifted to arg 1 and then parsed as an argument to `node` rather than to `addMamacMeters`.